runtime/connectors/s3/s3.go

```diff
@@ -76,3 +87,66 @@ type connector struct{}
 func (c connector) Spec() connectors.Spec {
 	return spec
 }
+
+// Consume will eventually be added to the interface
+func Consume(ctx context.Context, source *connectors.Source) (string, error) {
+	conf, _ := ParseConfig(source.Properties)
+
+	// The session the S3 Downloader will use
+	sess := session.Must(session.NewSession(&aws.Config{
```

Comment:
`Must` will panic if an error is returned ([see here](https://github.com/aws/aws-sdk-go/blob/d5e20b80023f98afe5a9200ffe5d026a020fa6ca/aws/session/session.go#L379)). Just handle and return the error from `session.NewSession` instead)